### PR TITLE
Removing hard coded xml tags

### PR DESCRIPTION
**Pull request by Drifftr**

### CODE REVIEW

File: plugins/org.wso2.developerstudio.humantaskeditor/HumanTaskEditor/hteditor.js [hunk 1]
```diff
@@ -910,7 +910,7 @@ function addPeopleAssignementNode(taskNode, xmlDom, assignmentName) {
             "http://docs.oasis-open.org/ns/bpel4people/ws-humantask/200803",
             "htd:argument");
         newArgument.setAttribute("name", "role");
-        newFrom.setAttribute("logicalPeopleGroup", "regionalClerks");
+        //newFrom.setAttribute("logicalPeopleGroup", "");
```
Comment (vinok88) [inline plugins/org.wso2.developerstudio.humantaskeditor/HumanTaskEditor/hteditor.js]: Remove commented codes.

File: plugins/org.wso2.developerstudio.humantaskeditor/HumanTaskEditor/hteditor.js [hunk 2]
```diff
@@ -1002,11 +1002,16 @@ function marshalPeopleAssignment(taskNode, peopleAssignmentName) {
         .getElementsByTagName(peopleAssignmentName).length != 0 && taskNode.getElementsByTagName("peopleAssignments")[0]
         .getElementsByTagName(peopleAssignmentName)[0]
         .getElementsByTagName("argument").length != 0 ){
-    if ($('#' + taskDivName + " input[name="+peopleAssignmentName+ "" + taskDivName + "]:checked").val() == peopleAssignmentName+ "Role")
+    if ($('#' + taskDivName + " input[name="+peopleAssignmentName+ "" + taskDivName + "]:checked").val() == peopleAssignmentName+ "Role"){
```
Comment (vinok88) [inline plugins/org.wso2.developerstudio.humantaskeditor/HumanTaskEditor/hteditor.js]: Need formatting. Add space between '+' marks.

File: plugins/org.wso2.developerstudio.humantaskeditor/HumanTaskEditor/hteditor.js [hunk 3]
```diff
@@ -1036,6 +1041,12 @@ function unmarshalPeopleAssignment(taskNode, peopleAssignmentName) {
                         taskNode.getElementsByTagName("peopleAssignments")[0]
                         .getElementsByTagName(peopleAssignmentName)[0]
                         .getElementsByTagName("argument")[0].childNodes[0].nodeValue.trim());
+                if(ownerType == "Role"){
+                    $('#' + taskDivName + " #"+peopleAssignmentName+"RoleLogicalGroup").val(
```
Comment (vinok88) [inline plugins/org.wso2.developerstudio.humantaskeditor/HumanTaskEditor/hteditor.js]: Need formatting.

File: plugins/org.wso2.developerstudio.humantaskeditor/HumanTaskEditor/hteditor.js [hunk 4]
```diff
@@ -1036,6 +1041,12 @@ function unmarshalPeopleAssignment(taskNode, peopleAssignmentName) {
                         taskNode.getElementsByTagName("peopleAssignments")[0]
```
Comment (himasha) [inline plugins/org.wso2.developerstudio.humantaskeditor/HumanTaskEditor/hteditor.js]: Have the tag names as constants.

File: plugins/org.wso2.developerstudio.humantaskeditor/HumanTaskEditor/hteditor.js (hunk 1 above)
Comment (Drifftr) [inline plugins/org.wso2.developerstudio.humantaskeditor/HumanTaskEditor/hteditor.js]: Fixed in https://github.com/wso2/devstudio-tooling-bps/pull/25

File: plugins/org.wso2.developerstudio.humantaskeditor/HumanTaskEditor/hteditor.js (hunk 2 above)
Comment (Drifftr) [inline plugins/org.wso2.developerstudio.humantaskeditor/HumanTaskEditor/hteditor.js]: Fixed in https://github.com/wso2/devstudio-tooling-bps/pull/25

File: plugins/org.wso2.developerstudio.humantaskeditor/HumanTaskEditor/hteditor.js (hunk 3 above)
Comment (Drifftr) [inline plugins/org.wso2.developerstudio.humantaskeditor/HumanTaskEditor/hteditor.js]: Fixed in https://github.com/wso2/devstudio-tooling-bps/pull/25

File: plugins/org.wso2.developerstudio.humantaskeditor/HumanTaskEditor/hteditor.js (hunk 4 above)
Comment (Drifftr) [inline plugins/org.wso2.developerstudio.humantaskeditor/HumanTaskEditor/hteditor.js]: Fixed in https://github.com/wso2/devstudio-tooling-bps/pull/25

File: plugins/org.wso2.developerstudio.humantaskeditor/src/org/wso2/developerstudio/humantaskeditor/wizards/HumanTaskExportWizard.java [hunk 5]
```diff
@@ -111,12 +111,12 @@ private void doFinish(IProgressMonitor monitor) throws Exception {
                     if (deployedZip.exists()){
                         return;
                     }else{
-                        throw new Exception("Could not copy the BPEL archive file to the given location.");
+                        throw new Exception("Could not copy the human task archive file to the given location.");
```
Comment (Niranjan-K) [inline plugins/org.wso2.developerstudio.humantaskeditor/src/org/wso2/developerstudio/humantaskeditor/wizards/HumanTaskExportWizard.java]: Why are we throwing generic Exception?

**Pull request merged**